app/controllers/question.rb

```diff
@@ -0,0 +1,42 @@
+get '/questions/new' do
+  redirect to "/login" unless logged_in?
+  @path = "/questions/new"
+  @method = "post"
```

Comment:
These two feel a little unnecessary to set in your controller rather than directly in the view when you need them, but I don't mind it too much.